lib/scholar/scaler/standard_scaler.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Scholar.Scaler.StandardScaler do
```

Comment:
Let's put this in the Preprocessing namespace:

```suggestion
defmodule Scholar.Preprocessing.StandardScaler do
```